### PR TITLE
Rename redis in valkey-cli file comments and prints.

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
Updated to Valkey in valkey-cli.c file's comments and prints.

* The output of valkey-cli --help
* The output of the cli built-in HELP command
* The prompt in interactive valkey-cli -s unixsocket
* The history file and the default rc file (changed filename)

### CODE REVIEW

File: src/valkey-cli.c [hunk 1]
```diff
@@ -75,9 +75,9 @@
 #define REDIS_CLI_KEEPALIVE_INTERVAL 15 /* seconds */
 #define REDIS_CLI_DEFAULT_PIPE_TIMEOUT 30 /* seconds */
 #define REDIS_CLI_HISTFILE_ENV "REDISCLI_HISTFILE"
-#define REDIS_CLI_HISTFILE_DEFAULT ".rediscli_history"
+#define REDIS_CLI_HISTFILE_DEFAULT ".valkeycli_history"
```
Comment (zuiderkwast) [inline src/valkey-cli.c]: Why not also rename the macros? We can remove the REDIS prefix on all of them i think.

```suggestion
#define CLI_HISTFILE_DEFAULT ".valkeycli_history"
```

File: src/valkey-cli.c [hunk 2]
```diff
@@ -3091,17 +3091,17 @@ version,tls_usage);
 "                     Use filename of \"-\" to write to stdout.\n"
 "  --functions-rdb <filename> Like --rdb but only get the functions (not the keys)\n"
 "                     when getting the RDB dump file.\n"
-"  --pipe             Transfer raw Redis protocol from stdin to server.\n"
+"  --pipe             Transfer raw Valkey protocol from stdin to server.\n"
```
Comment (zuiderkwast) [inline src/valkey-cli.c]: The protocol is called RESP. Let's use this word when talking about the protocol.

```suggestion
"  --pipe             Transfer raw RESP protocol from stdin to server.\n"
```

File: src/valkey-cli.c [hunk 3]
```diff
@@ -3091,17 +3091,17 @@ version,tls_usage);
...
 "                     no reply is received within <n> seconds.\n"
 "                     Default timeout: %d. Use 0 to wait forever.\n",
     REDIS_CLI_DEFAULT_PIPE_TIMEOUT);
     fprintf(target,
-"  --bigkeys          Sample Redis keys looking for keys with many elements (complexity).\n"
-"  --memkeys          Sample Redis keys looking for keys consuming a lot of memory.\n"
-"  --memkeys-samples <n> Sample Redis keys looking for keys consuming a lot of memory.\n"
+"  --bigkeys          Sample Valkey keys looking for keys with many elements (complexity).\n"
+"  --memkeys          Sample Valkey keys looking for keys consuming a lot of memory.\n"
+"  --memkeys-samples <n> Sample Valkey keys looking for keys consuming a lot of memory.\n"
```
Comment (zuiderkwast) [inline src/valkey-cli.c]: For these, i think it's better to just remove the word Valkey.

Sample keys looking for ....

**Review by zuiderkwast — COMMENTED** [review]
Nice!

I think we shall also backport some of this (the prompt in the interactive cli program) to our release 7.2.x. @valkey-io/core-team

File: src/valkey-cli.c (hunk 1 above)
Comment (Shivshankar-Reddy) [reply src/valkey-cli.c]: Thought will rename with all macro rename together and make as simple as related to only comments and logs. And now I changed macro as well and remaining macros will create another pr.

File: src/valkey-cli.c (hunk 2 above)
Comment (Shivshankar-Reddy) [reply src/valkey-cli.c]: updated accordingly. thanks!

File: src/valkey-cli.c (hunk 3 above)
Comment (Shivshankar-Reddy) [reply src/valkey-cli.c]: updated!

**Review by zuiderkwast — APPROVED** [review]
Great! Another pr for the macros sounds good.

**Review by enjoy-binbin — APPROVED** [review]

**Comment by zuiderkwast** [comment]
@valkey-io/core-team I have cherry-picked this to the 7.2 branch. (I did it after the rc1 release, so it will be in the next one.)

**Pull request merged**